sql/src/main/java/org/apache/druid/sql/calcite/planner/convertlet/DruidConvertletTable.java

```diff
@@ -44,6 +45,7 @@ public class DruidConvertletTable implements SqlRexConvertletTable
       ImmutableList.<DruidConvertletFactory>builder()
                    .add(CurrentTimestampAndFriendsConvertletFactory.INSTANCE)
                    .add(TimeInIntervalConvertletFactory.INSTANCE)
+                   .add(new NestedDataOperatorConversions.DruidJsonValueConvertletFactory())
```

Comment:
Any reason this isn't in a static field like the other ones?

Reply:
no reason, will change :+1: